packages/formstr-app/src/containers/CreateFormNew/components/QuestionCard/CardHeader.tsx

```diff
@@ -28,9 +26,66 @@ const CardHeader: React.FC<CardHeaderProps> = ({
   lastQuestion,
 }) => {
   const { MOBILE } = useDeviceType();
-  const { toggleSettingsWindow } = useFormBuilderContext();
+  const { toggleSettingsWindow, deleteQuestion, setQuestionIdInFocus } = useFormBuilderContext();
+  const [showUndo, setShowUndo] = useState(false);
```

Comment:
should delete button be it's own component, if not? this state woould be better called as showUndoDelete